transcom/mymove
# INT B-20113 refactor edit profile redux form

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-20113)

## Summary

Another lovely `redux-form` BL item. `EditProfile` was not being used anywhere in the code and was free to be deleted.

![Screenshot 2024-08-01 at 1 42 00 PM](https://github.com/user-attachments/assets/e490a6e5-4704-4999-b573-c9566c989fe9)

### How to test

1. Access MM as any role
2. Ensure you can still edit the customer profile in any capacity
